### PR TITLE
Update swim-lessons.md

### PR DESCRIPTION
**Pull request by ALFalls**
Edits @sspicer1 @SharonManyara

### CODE REVIEW

**Comment by jhutchings1** [comment]
👋🏻  Hey @ALFalls Just an FYI that the staging site only keeps the most recent branch available for testing. If you'll be making many updates in a day, I recommend putting them all on a single branch and testing them together.

File: _amenities/pools/swim-lessons.md [hunk 1]
```diff
@@ -9,30 +9,32 @@ show-title: true
 ---
 
 <div class="alert alert-warning">
-<strong>NOTE:</strong> Klahanie pools are open and swim lessons are being offered with limitations. 
+  
+ Klahanie pools are open, and swim lessons are available. For the safety of our residents and staff, we follow CDC guidelines for the prevention of COVID-19. 
+ 
+<strong>Quick Links</strong>
```
Comment (jhutchings1) [inline _amenities/pools/swim-lessons.md]: What are you trying to accomplish with this change? This moves the Quick links header out of the floating card on the right side of the page and puts it into the yellow alert banner.

**Comment by ALFalls** [comment]
> 👋🏻 Hey @ALFalls Just an FYI that the staging site only keeps the most recent branch available for testing. If you'll be making many updates in a day, I recommend putting them all on a single branch and testing them together.

@jhutchings1 Sure. Can you please send instructions for doing so or point me to a tutorial that gives steps? Thx.

**Review by SharonManyara — CHANGES_REQUESTED** [review]
Attempting to correct the "Note" code problem.

**Comment by jhutchings1** [comment]
If you look at the diff here, you'll see that there are still changes to the menu on line 16. You should put it back like it was to avoid changing that menu. https://github.com/Klahanie/klahanie/pull/97/files

Also, @SharonManyara , it looks like you selected `request changes`. Please be aware that that choice blocks the pull request until you change your vote to `approve`. https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews

**Review by SharonManyara — APPROVED** [review]

**Review by jhutchings1 — APPROVED** [review]

**Pull request merged**